### DISCUSSION TITLE
Workflow badge "no status"

### DISCUSSION THREAD

**Discussion by inwardmovement** [community/community#25097] [Actions]
<p>I followed the <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository" rel="nofollow noopener">documentation</a>, but I have one of two badges that shows “no status”. Is this a bug?</p>
<p>Repo: <a href="https://github.com/algosphere/algo-website" rel="nofollow noopener">https://github.com/algosphere/algo-website</a></p>
<p>Code:</p>
<pre><code>![ci-cd-develop](https://github.com/algosphere/algo-website/workflows/ci-cd-develop/badge.svg?branch=develop)
![ci-cd-master](https://github.com/algosphere/algo-website/workflows/ci-cd-master/badge.svg?branch=master)
</code></pre>
<p>It’s ok for <em>develop</em> but “no status” for <em>master</em>.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>I checked your master/ develop branch commits history. It seems that when you merge from develop branch to master branch, it used fast-forward merge.</p>
<p><img src="https://user-images.githubusercontent.com/24806478/181088757-7c56e2fe-73e9-4908-84a2-61be3292e251.png" alt="fast-forward.png" data-base62-sha1="7Fsi357uEYjpwKVjaAqSkJy6Jio" width="617" height="301"></p>
<p>This is a known issue. Please check the previous tickets:</p>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.community%2Ft5%2FGitHub-Actions%2FWorkflow-badge-always-quot-no-status-quot-when-limited-to-branch%2Ftd-p%2F40051&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7C0ba35bebc9ab4d8c28bf08d78db96a11%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637133699796476393&amp;sdata=HuCZ1LfGIZo3w%2BTyQSuClbkcwwXUq46YFAkyXyNFUuw%3D&amp;reserved=0" rel="nofollow noopener">https://github.community/t5/GitHub-Actions/Workflow-badge-always-quot-no-status-quot-when-limited-to-branch/td-p/40051</a></p>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.community%2Ft5%2FGitHub-Actions%2FActions-Reporting-Wrong-Branch-on-Merges%2Ftd-p%2F36966&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7C0ba35bebc9ab4d8c28bf08d78db96a11%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637133699796486381&amp;sdata=0G7HujUizlPK0oZvQ0TNZknFaH%2B%2Bg2fjXS%2BGwT5HDg0%3D&amp;reserved=0" rel="nofollow noopener">https://github.community/t5/GitHub-Actions/Actions-Reporting-Wrong-Branch-on-Merges/td-p/36966</a></p>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.community%2Ft5%2FGitHub-Actions%2FWrong-branch-displayed-for-workflow%2Fm-p%2F37985%23M3178&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7C0ba35bebc9ab4d8c28bf08d78db96a11%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637133699796486381&amp;sdata=DtfzPlxg2zxOJiLWAUZs5RA4tONZnEYS%2FmAGH9W%2BPhY%3D&amp;reserved=0" rel="nofollow noopener">https://github.community/t5/GitHub-Actions/Wrong-branch-displayed-for-workflow/m-p/37985#M3178</a></p>
<p>I would encourage you to merge using pull request , when it is merged, there will be a merged commit push to master. This could avoid this issue.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/64e814ba5d13fa0b747e47271e3dbcf00d0516dc.png" data-download-href="/uploads/short-url/eoEYEiUlwXtZgIfVGhbt9MvmfeY.png?dl=1" title="merge commit.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181088759-e69fedc9-81df-4a8d-92e9-adb31dae507c.png" alt="merge commit.png" data-base62-sha1="eoEYEiUlwXtZgIfVGhbt9MvmfeY" width="690" height="160" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/64e814ba5d13fa0b747e47271e3dbcf00d0516dc_2_690x160.png, https://aws1.discourse-cdn.com/github/original/2X/6/64e814ba5d13fa0b747e47271e3dbcf00d0516dc.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/64e814ba5d13fa0b747e47271e3dbcf00d0516dc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/64e814ba5d13fa0b747e47271e3dbcf00d0516dc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">merge commit.png</span><span class="informations">1030×239 14.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by inwardmovement** [comment]
<p>Ok thanks <a class="mention" href="/yanjingzhu">@yanjingzhu</a>, I look forward to a fix then.</p>
<p>Thanks for the recommandation though!</p>
